eng/common/pipelines/templates/jobs/archetype-sdk-tests.yml

```diff
@@ -0,0 +1,42 @@
+parameters:
```

Comment:
What is your plan for this file? It looks like a sample as opposed to a template for consumption

Reply:
It is. See my [above comment](https://github.com/Azure/azure-sdk-tools/pull/1429#discussion_r579325972). The archetype-sdk-tests.yml job will never be able to be standardized because the implementation differs from repo to repo. So our options are to have a non-functional matrix generation template as a reference, or to have this file which acts as the "common/tools implementation of the main job."

Reply:
Ok is the plan then to mutate this into the common one that everyone uses? Or will this always remain a sample? If it remains a sample I think we should consider moving it out of these template directories and into the samples folder.

Reply:
It will always remain a sample, because the job implementation is specific to the repository. If we want to move this file out of the template directory, but still have the *generate.yml be shared, then we will need to pass in the template path as a parameter, because the repositories will need to have the *generate.yml file point to their job implementation in `eng/pipelines/templates/jobs`.

Reply:
Updated to parameterize the job path, which I think is right anyway (that way we don't necessarily have to standardize the file names across the repos, although I think we should).

Reply:
Cool yeah I like passing it as a parameter.